### DISCUSSION TITLE
Collaborative folder

### DISCUSSION THREAD

**Discussion by thelinuxlich** [massCodeIO/massCode#116] [Ideas]
It would be amazing if we could set up a folder shared between peers, probably stored separately in a remote origin (or decentralized, maybe using GunDB)

**Comment by antonreshetov** [comment]
In general, I don't look at colaborative editing. I honestly don't understand what this functionality is for, for a personal notebook, which is essentially what massCode is.

**Reply by thelinuxlich** [reply]
Example: a folder for company-specific snippets, other folders would be particular.

**Reply by antonreshetov** [reply]
And why share access to specific snippets, which in addition can someone from the participants to change or delete without your knowledge?

**Reply by antonreshetov** [reply]
I also think that this kind of functionality does not fit into the open source model. It's more of an enterprise solution that the user has to pay for.

**Reply by thelinuxlich** [reply]
well, ideas for you to monetize upon :), but company could be an opensource org too

**Reply by thelinuxlich** [reply]
and about losing knowledge, it could support versioning

**Reply by antonreshetov** [reply]
> well, ideas for you to monetize upon :), but company could be an opensource org too

massCode needs to gain fame to begin with ))

**Reply by antonreshetov** [reply]
> and about losing knowledge, it could support versioning

Sure, but all this complicates the idea of massCode. Maybe in the future I can use your suggestion, thanks!

**Reply by thelinuxlich** [reply]
no problem! Love this project anyway!
